### ISSUE TITLE
STDOUT Support

### ISSUE THREAD

**Issue by jrlcloud** [bwa-mem2/bwa-mem2#90]
Hi there!
For bwa mem, I can get the output results from STDOUT by default if I don't specify the output SAM file name.
This feature helps when I need to pass the output alignment data to downstream analysis through pipe for the benefits of I/O reduction.
I failed to find the same feature in bwa-mem2, is there any hidden parameter or usage to achieve this?

**Comment by keiranmraine** [comment]
`bwa-mem2` outputs to STDOUT in same way as old bwa.  Providing an example command is always helpful in diagnosing usage issues.  Please be aware that memory requirements are currently much higher, you may be hitting a different problem.

**Comment by jrlcloud** [comment]
Oh thanks for the reminder on memory requirements, it sure solve my problem!
The command line (same as my bwa-mem cmd) was fine, its just I submitted the bwa-mem2 job with a same memory requirements for bwa-mem which was insufficient (as you mentioned)，and the job was killed at the index loading stage which I thought was a STDOUT issue.
The result is OK after I raised the memory requirements for the job, I'm closing this issue now, thanks!
